src/platforms/hosted/cmsis_dap.c

```diff
@@ -87,6 +87,9 @@ typedef enum cmsis_type {
 typedef struct hid_device_info hid_device_info_s;
 #endif
 
+/*- Forward declaration -----------------------------------------------------*/
```

Comment:
Drop the comment, it's not adding anything (this file still needs style fixes). Also drop the `const` marking on the data parameter in this declaration (clang-tidy lint)

Reply:
Done. I did not modify the actual implementation, to minimize the diff.